src/org/traccar/FilterHandler.java

```diff
@@ -145,8 +145,17 @@ private boolean skipLimit(Position position, Position last) {
         return false;
     }
 
-    private boolean skipAlarms(Position position) {
-        return skipAlarms && position.getAttributes().containsKey(Position.KEY_ALARM);
+    private boolean skipAttributes(Position position) {
+        if (skipAttributes) {
+            String attributesString = Context.getIdentityManager().lookupAttributeString(
+                    position.getDeviceId(), "filter.skipAttributes", "", true);
+            for (String attribute : attributesString.split("[ ,]")) {
```

Comment:
I think we need to make it more efficient than parsing string every time. Maybe parse it into a list once when we create handler.

Reply:
We are looking up in devices tree, not only in config. It is impossible to "pre-parse"

Reply:
Good point. I forgot about it.